src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,36 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: 7 * $cell-size;
```

Comment:
here you have to add cell gap of` 1px` between each cell that will be `$cell-gap * 6`
and also add calendar padding of 10px for both sides `$calender-padding * 2`